test/lib/file_test.exs

```diff
@@ -247,8 +248,24 @@ defmodule Nacha.FileTest do
     end
 
     test "ccd-debit" do
-      assert {:ok, _file} =
+      assert {:ok, file} =
                NachaFile.parse("./test/fixtures/achfiles/ccd-debit.ach")
     end
+
+    test "ppd-debit" do
+      assert {:ok, _file} =
```

Comment:
Are you going to add tests testing the actual results returned from parsing?

Reply:
Yes. Next PR!